recipe/meta.yaml

```diff
@@ -24,9 +24,10 @@ requirements:
   host:
     - python
     - pip
-    - cython >=0.28.3
+    - cython >=0.29.29
     - numpy
     - libgdal
+    - wheel
   run:
     - python
     - setuptools
```

Comment:
`click` should be `click>=4.0`, see https://github.com/Toblerity/Fiona/blob/bae44db5845955870a850fb53f67c9a4ab4c55d4/setup.py#L288